crates/bevy_ui/src/render/ui.wgsl

```diff
@@ -27,9 +72,23 @@ var sprite_texture: texture_2d<f32>;
 @group(1) @binding(1)
 var sprite_sampler: sampler;
 
+fn distance_round_border(point: vec2<f32>, size: vec2<f32>, radius: f32) -> f32 {
+    var q = abs(point) - size + radius;
+    return min(max(q.x, q.y), 0.0) + length(max(q, vec2<f32>(0.0))) - radius;
+}
 @fragment
 fn fragment(in: VertexOutput) -> @location(0) vec4<f32> {
     var color = textureSample(sprite_texture, sprite_sampler, in.uv);
     color = in.color * color;
+
+    if (in.radius > 0.0 || in.border_width > 0.0) {
+        var distance = distance_round_border(in.point, in.size * 0.5, in.radius);
+
+        var inner_alpha = 1.0 - smoothstep(0.0, 0.0, distance);
```

Comment:
```suggestion
        var inner_alpha = 1.0 - smoothstep(0.0, 1.0, distance);
```

I believe this should be 1.0, otherwise you're not getting any anti-aliasing.

Reply:
![a button with the title "Button" showing pixelated edges](https://github.com/bevyengine/bevy/assets/66000635/7994e4f7-5c2e-460f-a707-85fc5858d7fe)

Ah, I see! 

![an anti-aliased button with 25px corner radius](https://github.com/bevyengine/bevy/assets/66000635/d8e47d68-f874-493d-9180-ae4f523e9be7)
![an anti-aliased button with 15px corner radius](https://github.com/bevyengine/bevy/assets/66000635/a8db4587-d07e-48f5-87a1-b3e40afeeeb3)

Though, it looks a bit cool, if only it's more consistent between all corners.